README.md

```diff
@@ -66,7 +66,6 @@ and health-checking mechanisms.
   On CentOS 7, if the facter version is less than 3.14 or not installed, then run the following commands to update/install it:
   
   ```sh
-  sudo yum erase -y $(rpm -q --whatprovides $(readlink -f /usr/bin/facter)) || sudo rm -fv /usr/bin/facter
```

Comment:
I think this was done for specific case of existing ver less that 3.14.
I think @pavankrishnat , when was testing this particular case, he was facing some issue, that is why I guess we had added removal part here. @pavankrishnat can confirm if my understanding is correct.

@andriytk ,
Just to confirm, was this specific case tested, where we don't erase existing ver which is less than 3.14 and just run next steps?

Reply:
@suvratjoshi/ @andriytk 
I added this line of code to fix #1779

Reply:
Yes, I tested it. The yum command deletes motr rpm! And if we don't do it, `ln -sf` cmd just replaces facter at `/usr/bin` with the version from puppet. Hare works fine then.

The problem with the puppet package is that it does not declare that it provides facter, so when we erase facter rpm - it affects motr, as motr rpm has a dependency on facter.

This ugly way of solving the problem will be gone in CentOS 8, which has 3.14 facter version rpm in the base repo - the one we need.